### PR TITLE
[experimental assets] remove misleading Squoosh message

### PR DESCRIPTION
**Pull request by sarah11918**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

Docs incorrectly describe Squoosh as working in "all JavaScript environments" when there are, in fact, limitations. This removes that statement, and makes a slightly stronger push for installing sharp in a Node environment.

### CODE REVIEW

File: src/content/docs/en/guides/assets.mdx [hunk 1]
```diff
@@ -336,9 +336,9 @@ It returns an object with the following properties:
 
 ## Using sharp
 
-By default, Astro uses [Squoosh](https://github.com/GoogleChromeLabs/squoosh) to transform your images. This supports all JavaScript environments.
+By default, Astro uses [Squoosh](https://github.com/GoogleChromeLabs/squoosh) to transform your images.
 
-If you're deploying to a Node environment, you may want to use [sharp](https://github.com/lovell/sharp), which offers faster performance but requires Node. To use sharp, first install it:
+If you're deploying to a Node environment, we recommend using [sharp](https://github.com/lovell/sharp), which offers faster performance but requires Node. To use sharp, first install it:
```
Comment (delucis) [inline src/content/docs/en/guides/assets.mdx:341]: Is it worth mentioning static sites separately? The “Node environment” is specifically relevant to SSR deployments I guess (as you _always_ need a Node environment for the initial build).
```suggestion
If you're building a static site or deploying an SSR site to a Node environment, we recommend using [sharp](https://github.com/lovell/sharp), which offers faster performance but requires Node. To use sharp, first install it:
```

**Review by delucis — COMMENTED** [review]
Looks good Sarah. Just one comment!

**Review by delucis — APPROVED** [review]
LGTM! 🚀

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**